nvtabular/framework_utils/torch/utils.py

```diff
@@ -17,6 +17,63 @@
 import torch
 
 
+class FastaiTransform:
+    def __init__(self, dataloader):
+        self.data = dict_transform(dataloader)
+
+    def transform(self, batch):
+        concats = []
+        for columns in [self.data.cats, self.data.conts]:
+            # cols = [v for k,v in batch[0] if k in columns and isinstance(v, torch.Tensor)]
+            cols = []
+            for k, v in batch[0].items():
+                if k in columns and not isinstance(v, tuple) and isinstance(v, torch.Tensor):
+                    cols.append(v)
+            concats.append(torch.cat(cols, axis=1))
+        return (
+            concats[0].to("cuda", dtype=torch.long),
+            concats[1],
+            batch[1].to("cuda", dtype=torch.long),
+        )
+
+
+class dict_transform:
```

Comment:
Why is this lower snake case? 

```suggestion
class DictTransform:
```